code/game/objects/items/weapons/implants/implanter.dm

```diff
@@ -57,22 +58,14 @@
 
 /obj/item/weapon/implanter/New()
 	..()
+	if(imp_type)
+		imp = new imp_type(src)

```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#prefer-initialize-over-new-for-atoms